### PR TITLE
Fixed: Healthchecks not firing due to missing IProvideHealthCheckWith…

### PR DESCRIPTION
**Pull request by bakerboy448**
…Message

#### Database Migration
NO

#### Description
title
per Ta it's in Readarr/Lidarr and not Radarr

#### Todos
- Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

**Comment by Qstick** [comment]
Isn't there an actual frontend health check that goes with this?

**Comment by bakerboy448** [comment]
> Isn't there an actual frontend health check that goes with this?

defer to @ta264 as he's who noticed it was missing from Radarr via https://github.com/Readarr/Readarr/pull/1328

**Review by ta264 — CHANGES_REQUESTED** [review]
I think you also need to edit the interface here: https://github.com/Radarr/Radarr/blob/093e076db06645106c5bc929b3b6d336308bd5dc/src/NzbDrone.Core/HealthCheck/Checks/RemotePathMappingCheck.cs#L25
To be
```IProvideHealthCheckWithMessage```

This should then hook up this function: https://github.com/Radarr/Radarr/blob/093e076db06645106c5bc929b3b6d336308bd5dc/src/NzbDrone.Core/HealthCheck/Checks/RemotePathMappingCheck.cs#L112

which right now is totally useless

**Comment by bakerboy448** [comment]
> I think you also need to edit the interface here:
> 
> https://github.com/Radarr/Radarr/blob/093e076db06645106c5bc929b3b6d336308bd5dc/src/NzbDrone.Core/HealthCheck/Checks/RemotePathMappingCheck.cs#L25
> 
> 
> To be
> `IProvideHealthCheckWithMessage`
> This should then hook up this function:
> 
> https://github.com/Radarr/Radarr/blob/093e076db06645106c5bc929b3b6d336308bd5dc/src/NzbDrone.Core/HealthCheck/Checks/RemotePathMappingCheck.cs#L112
> 
> which right now is totally useless

think i did that right / understood you right :)

File: src/NzbDrone.Core/HealthCheck/HealthCheckService.cs [hunk 1]
```diff
@@ -138,11 +148,12 @@ public void HandleAsync(IEvent message)
                 if (eventDrivenHealthCheck.ShouldExecute(message, previouslyFailed))
                 {
                     filteredChecks.Add(eventDrivenHealthCheck.HealthCheck);
+                    continue;
```
Comment (bakerboy448) [inline src/NzbDrone.Core/HealthCheck/HealthCheckService.cs:151]: I think we can remove this as it's redundant?

**Pull request closed without merging**